Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:8-alpine
```

Comment:
I would recommend using the latest possible version of node unless really needed otherwise.

Reply:
@smoke At the moment project can not built successfully that is why I am using nodejs 8 version, because with nodejs 10 and 11 'npm install' returns errors.